lms/static/sass/stanford/course/_sneakpeek_courseware.scss

```diff
@@ -0,0 +1,18 @@
+.sneakpeek-warning {
```

Comment:
We can't, because the course overview page (the new page that is the issue in the first place for sneak-peek) does NOT have a body class to grab on to.